docs/proposals/pepxxx_wheel_variant_support.md

```diff
@@ -454,6 +454,21 @@ USE flags have boolean values. This also has the downside that multiple flags ne
 
 <TO BE ADDED>: [https://wiki.debian.org/CategoryMultiarch](https://wiki.debian.org/CategoryMultiarch)
 
+## Reference implementations
+
+The [variantlib](https://github.com/wheelnext/variantlib) project contains a reference implementation illustrating how
```

Comment:
I get that you're just moving the text but perhaps it should be something more general, like that it contains a reference implementation of pretty much everything.

Then, giving very specific examples how to generate index doesn't really fit this section.

Reply:
Can you describe the core features of variantlib that we want to present? It doesn't have a Readme, so I've done some guessing based on the subcommands.

Reply:
I think the key file is https://github.com/wheelnext/variantlib/blob/main/variantlib/api.py, i.e. it has reference implementation of the protocol, querying plugins, variant metadata reading and writing, variant filtering and ordering, validation, preparing environment marker values. Basically, it's building blocks for build backends and package managers.